tests/mock_data.py

```diff
@@ -87,7 +91,7 @@
         "branch": "main",
         "commit_sha": "8cd58eb160014c91e4f181562352" "c693d3442c52",
     },
-    "dockerhub_token": "dockerhub token",
+    "dockerhub_token": "dXNlcjpwYXNzd2Q=",
```

Comment:
Is that a real token? If so, this really shouldn't be pushed and you should try to remove all traces from it from the repo.

Reply:
(and instead use a GH Actions secret for the CI and an env variable or file for development)

Reply:
no this is just `user:passwd`

Reply:
```